CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # React-Live Changelog
 
+## 4.0.0
+
+* Migrated codebase over to TypeScript
+* Added support for TypeScript-based components in the React Live editor
+
```

Comment:
I believe this should be auto-generated from your `.changelog/*.md` file, correct? I'd probably just add another changeset record for `Added support for TypeScript-based components in the React Live editor`, and then remove this change and let changeset do its thang.

Reply:
Sounds good. I think I did this as muscle memory since we didn't have changesets before.